index.md

```diff
@@ -0,0 +1,12 @@
+
+# List of the artists index z
+
+- [Zarahasina](zarahasina.md)
+- [Zafindraony](zafindraony.md)
+- [Zeze Mahanoro](zeze-mahanoro.md)
+
+- [Zakaria](zakaria.md)
+
+- [Zarah](zarah.md)
+
+Go back to the artist index
```

Comment:
This text looks good, but it needs to be a link to the root index.md file that Sugi and Clopedia are adding.